client/main.js

```diff
@@ -379,6 +379,8 @@ $(document).ready(function() {
   $('#nav-wiki-btn').on('click', toggleWiki);
 
   $('.wiki-aside-action-collapse').on('click', collapseWiki);
+  $('#wikiFrame').attr('src', 'http://freecodecamp.github.io' + window.location.pathname);
```

Comment:
it should go to whatever is typed in the url bar. if it doesn't, it'll default to root, I believe